jobs/payment-jobs/tasks/ejv_payment_task.py

```diff
@@ -288,3 +286,158 @@ def _get_invoices_for_payment(cls, account_id: int) -> List[InvoiceModel]:
+                    jv_params.effective_date,
+                    f"{line.invoice_id:<110}",
+                    jv_params.journal_name,
+                    line_total,
+                    line_number,
+                    "C" if not invoice_data.is_jv_reversal or not invoice_data.is_partial_refund else "D",
+                )
+
+                # Debit from GOV ACCOUNT GL for a transaction (non-reversal)
+                line_number += 1
+                control_total += 1
+                # If it's normal payment then the Gov account GL goes as Debit,
+                # else it goes as Credit as we need to credit the fund back to ministry.
+                account_jv += cls.get_jv_line(
+                    batch_type,
+                    jv_params.debit_distribution,
+                    jv_params.description,
+                    jv_params.effective_date,
+                    f"{line.invoice_id:<110}",
+                    jv_params.journal_name,
+                    line_total,
+                    line_number,
+                    "D" if not invoice_data.is_jv_reversal or not invoice_data.is_partial_refund else "C",
+                )
+
+            if line_service_fee and line_service_fee > 0:
+                service_fee_distribution_code = DistributionCodeModel.find_by_id(
+                    line_distribution_code.service_fee_distribution_code_id
+                )
+                total += line_service_fee
+
+                # Credit to BCREG GL for a transaction (non-reversal)
+                line_number += 1
+                control_total += 1
+                account_jv += cls.get_jv_line(
+                    batch_type,
+                    cls.get_distribution_string(service_fee_distribution_code),
+                    jv_params.description,
+                    jv_params.effective_date,
+                    f"{line.invoice_id:<110}",
+                    jv_params.journal_name,
+                    line_service_fee,
+                    line_number,
+                    "C" if not invoice_data.is_jv_reversal or invoice_data.is_partial_refund else "D",
+                )
+
+                # Debit from GOV ACCOUNT GL for a transaction (non-reversal)
+                line_number += 1
+                control_total += 1
+                account_jv += cls.get_jv_line(
+                    batch_type,
+                    jv_params.debit_distribution,
+                    jv_params.description,
+                    jv_params.effective_date,
+                    f"{line.invoice_id:<110}",
+                    jv_params.journal_name,
+                    line_service_fee,
+                    line_number,
+                    "D" if not invoice_data.is_jv_reversal or invoice_data.is_partial_refund else "C",
+                )
+
+        return total, line_number, control_total, account_jv
+
+    @classmethod
+    def _calculate_line_fees(cls, line: PaymentLineItemModel, invoice_data: InvoiceData) -> tuple[float, float]:
```

Comment:
one payment_line_item might has multiple partial refunds:
<img width="612" alt="image" src="https://github.com/user-attachments/assets/d74d5d4a-4de5-41de-91ec-4127d079ba78" />

Reply:
I think we discussed this already, Base fees, priority fees, future_effective fees are all inside of `total` in payment_line_item